### PR TITLE
Update to MSAL 4.61 / net7-xyz

### PR DESCRIPTION
**Pull request by bgavrilMS**
Update to MSAL 4.61.0 and to net7

### CODE REVIEW

File: MauiAppB2C/MauiB2C.csproj [hunk 1]
```diff
@@ -1,10 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-android;net6.0-ios</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net6.0-windows10.0.19041.0</TargetFrameworks>
-		<!-- Uncomment the following line to build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
-		<!-- <TargetFrameworks>$(TargetFrameworks);net6.0-tizen</TargetFrameworks> -->
+		<TargetFrameworks>net7.0-android;net7.0-ios</TargetFrameworks>
```
Comment (pmaytak) [inline MauiAppB2C/MauiB2C.csproj:4]: Update to NET 8 since it's LTS?

File: MauiAppB2C/MauiB2C.csproj [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net7.0-windows10.0.19041.0</TargetFrameworks>
```
Comment (pmaytak) [inline MauiAppB2C/MauiB2C.csproj:5]:
```suggestion
		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net7.0-windows</TargetFrameworks>
```

Comment (bgavrilMS) [reply MauiAppB2C/MauiB2C.csproj:5]: I think this is how VS creates the app by default. Probably to give access to Windows SDK apis

File: MauiAppB2C/MauiB2C.csproj (hunk 1 above)
Comment (bgavrilMS) [reply MauiAppB2C/MauiB2C.csproj:4]: Hey @pmaytak , .NET 7 is still supported. I don't have time to retest everything with .net8. I think it should be separate effort.

**Review by pmaytak — APPROVED** [review]

File: MauiAppB2C/MauiB2C.csproj (hunk 2 above)
Comment (pmaytak) [reply MauiAppB2C/MauiB2C.csproj:5]: Ah, yea. I remember. MAUI Windows is technically only above that Win version, not earlier.

**Pull request merged**